src/utils/module.c

```diff
@@ -70,12 +71,16 @@ gboolean bd_utils_have_kernel_module (const gchar *module_name, GError **error)
     }
 
     path = kmod_module_get_path (mod);
-    have_path = (path != NULL) && (g_strcmp0 (path, "") != 0);
+    module_present = (path != NULL) && (g_strcmp0 (path, "") != 0);
```

Comment:
I like the old name of the variable better. `module_present` sounds to me like it's either loaded/available or builtin.